README.md

```diff
@@ -61,6 +61,10 @@ make/model/device ID matching, ...
 
 For compiling and using this package see the INSTALL file.
 
+JPEG-XL support is optionally available. It is activated if the JPEG-XL
+library (libjxl) and its header files are installed when building
+libcupsfilters.
+
 Report bugs to
```

Comment:
This is not needed, as we do not document which formats are (optionally) supported in this file.  Only place where this is treated is INSTALL.md where you did it correctly.